tests/test_openclhpp.cpp

```diff
@@ -4336,4 +4337,52 @@ void testTemplateGetImageRequirementsInfo()
 void testTemplateGetImageRequirementsInfo() {}
 #endif // cl_ext_image_requirements_info
 
+static cl_mem clCreateFromGLBuffer_testgetObjectInfo(cl_context context,
+                                                     cl_mem_flags flags,
+                                                     cl_GLuint bufobj,
+                                                     cl_int *errcode_ret,
+                                                     int num_calls)
+{
+    (void)num_calls;
+
+    TEST_ASSERT_EQUAL_PTR(make_context(0), context);
+    TEST_ASSERT_EQUAL(0, flags);
```

Comment:
Could we also test that the passed-in `bufobj` matches what was passed to the `cl::BufferGL` constructor?

Reply:
Test added

Reply:
Thanks, I don't think we need the `switch (num_calls)` though, since we're only creating one `cl::BufferGL`.

To be clear, I think only creating one `cl::BufferGL` is fine, so the fix is to remove the `switch (num_calls)`, not to create another `cl::BufferGL`.

Reply:
Removed switch